lib/js_locale_helper.rb

```diff
@@ -212,17 +213,15 @@ def self.output_extra_locales(bundle, locale)
 
     return "" if translations.blank?
 
-    <<~JS
-      if (!I18n.extras) {
-        I18n.extras = {}
-      }
-
-      if (!I18n.extras["#{locale}"]) {
-        I18n.extras["#{locale}"] = {};
-      }
+    output = +"if (!I18n.extras) { I18n.extras = {}; }"
```

Comment:
What does the `+` do in this case?

Reply:
`""` strings are frozen by default, so appending to it (`<<`) would fail with `FrozenError - can't modify frozen String: "…"`

Adding `+` makes the string dynamic. 🙂